### DISCUSSION TITLE
Filer logs can auto delete when expired?

### DISCUSSION THREAD

**Discussion by creeew** [seaweedfs/seaweedfs#3269] [Q&A] [answered]
I found new filer logs will be store in filer store and object storage when file changes. And they store in the same log file if the logs produced at the same minute.
We all know filer logs are quite important for filer sync and backup. But looks like there are no features about deleting those "expired" logs. And logs would fiercely increased if we have many modify operations in a long time.

Does SeaweedFS have any features about auto deleting log files for releasing the log storage cost?

**Comment by guol-fnst** [comment]
linux logrotate

**Reply by creeew** [reply]
Firend, I'm talking about filer's log which contain different file change metadatas at the same minute. Not code logs by glog.

**Comment by chrislusf** [comment, marked as answer]
There is a plan to compact the logs regularly. Not started yet.
